nikola/image_processing.py

```diff
@@ -139,8 +139,8 @@ def resize_image(self, src, dst, max_size, bigger_panoramas=True, preserve_exif_
             else:
                 im.save(dst, icc_profile=icc_profile)
         except Exception as e:
-            self.logger.warn("Can't process {0}, using original "
-                             "image! ({1})".format(src, e))
+            self.logger.warning("Can't process {0}, using original "
+                                "image! ({1})".format(src, e))
```

Comment:
How about monkeypatching `notice` and `warn` methods to the loggers returned by `utils.get_logger`, so that 3rd party plugins using these don't simply break? Or should this PR only go into a new major version?

Reply:
`notice` is already monkeypatched. `warn` is supported in `logging` but deprecated.

Reply:
Ah, I overlooked it. Thanks!